setup.cfg

```diff
@@ -54,7 +54,9 @@ exclude =
     *.tests
 
 [options.package_data]
-tensorbay = opendataset/*/catalog*.json
+tensorbay =
+    opendataset/*/catalog*.json
+    py.typed
```

Comment:
it seems that mypy would check types even without adding `py.typed` in `option.package_data`

Reply:
if the `py.typed` is not added in to `option.package_data`, it will not appear in tensorbay wheel package.
which means if customs install tensorbay by `pip install`, `py.typed` will not appear in the package they installed.